js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -73,6 +75,11 @@ export default function PaidAdsSetupSections( {
 		return resolveInitialPaidAds( startingPaidAds );
 	} );
 
+	const { data: budgetData } = useFetchBudgetRecommendationEffect(
```

Comment:
@dsawardekar We no longer have `countryCode` as part of the `paidAds` object now. Instead we have the `countryCodes` prop which you can use:
```suggestion
const { data: budgetData } =
		useFetchBudgetRecommendationEffect( countryCodes );
```

Reply:
@asvinb Updated to use the prop.